GREEDY/ReceiptCreatings/AutoCorrect.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using GREEDY.Models;
+using NHunspell;
+using System.IO;
+using System.Linq;
+
+namespace GREEDY.ReceiptCreatings
+{
+    class AutoCorrect
+    {
+        private static LevenshteinDistance LevenshteinDistance => new LevenshteinDistance();
```

Comment:
Same thing as in ReceiptService, new instance will be created each time the property is referenced